app/controllers/people_controller.rb

```diff
@@ -102,8 +104,8 @@ def edit
   end
```

Comment:
On line 103, we should make sure that phones and emails are handed into the form. I believe we are broken right now.